core/src/components/toast/toast.tsx

```diff
@@ -77,7 +78,7 @@ export class Toast implements ComponentInterface, OverlayInterface {
    * How many milliseconds to wait before hiding the toast. By default, it will show
    * until `dismiss()` is called.
    */
-  @Prop() duration = 0;
+  @Prop() duration = config.get('toastDuration', 0);
```

Comment:
Can we also add a test for this? You could write a test that sets `window.Ionic = { config: { toastDuration: 5000 } }` and checks that the `duration` property on `ion-toast` defaults to `5000`. We should also check to make sure that it defaults to `0` when the config option is not set.

Reply:
Yes of course. Added 2 tests.